build/gulp/tasks/test-projects.tsx

```diff
@@ -16,11 +17,33 @@ const log = (context: string) => (message: string) => {
   console.log('='.repeat(80))
 }
 
-export const publishPackage = async () => {
-  const filename = tmp.tmpNameSync({ prefix: 'stardust-', postfix: '.tgz' })
-  await sh(`yarn pack --filename ${filename}`)
+const installStardustPackages = async (tmpDirectory: string) => {
```

Comment:
and, frankly, it seems that this function has more responsibilities than it should - we could split it on the following ones, so that it will be easier to read the sequence of actions in the test method itself (client of this function), something like:

```
installStardustPackages ->
  packStardustPackages
  addResolutionPathsForStardustPackages
  `yarn add @stardust-ui/react` // as, actually, this is the only thing that we would like to reference as dependency
```